proto/protocolapis/astria/protocol/transaction/v1/action.proto

```diff
@@ -211,6 +212,35 @@ message BridgeSudoChange {
   string fee_asset = 4;
 }
 
+message BridgeTransfer {
+  // the address of the bridge account to transfer to
+  astria.primitive.v1.Address to = 1;
+  // the amount to transfer
+  astria.primitive.v1.Uint128 amount = 2;
+  // the asset used to pay the transaction fee
+  string fee_asset = 3;
+  // the address on the destination chain which
+  // will receive the bridged funds
+  string destination_chain_address = 4;
+
+  // The memo field can be used to provide unique identifying additional
+  // information about the bridge unlock transaction.
+  string memo = 5;
+  // the address of the bridge account to transfer from
+  astria.primitive.v1.Address bridge_address = 6;
+  // The block number on the rollup that triggered the transaction underlying
+  // this bridge unlock memo.
+  uint64 rollup_block_number = 7;
+  // An identifier of the original rollup event, such as a transaction hash which
+  // triggered a bridge unlock and is underlying event that led to this bridge
+  // unlock. This can be utilized for tracing from the bridge back to
```

Comment:
```suggestion
  // triggered a bridge transfer and is underlying event that led to this bridge
  // transfer. This can be utilized for tracing from the bridge back to
```